include/aws/testing/aws_test_harness.h

```diff
@@ -35,6 +36,8 @@ the AWS_UNSTABLE_TESTING_API compiler flag
 #define AWS_TESTING_REPORT_FD stderr
 #endif
 
+static struct aws_mutex mutex;
```

Comment:
I think I would give this a more descriptive name than "mutex"

Reply:
sure, I'll rename it something like, allocator_lock.